src/scripts.js

```diff
@@ -21,6 +21,10 @@ const cardButtonConditionals = (event) => {
     domUpdates.populateCards(domUpdates.cookbook.recipes);
   } else if (event.target.classList.contains('close-button')) {
     domUpdates.removeFromToCook(event);
+  } else if (event.target.classList.contains('have-cooked')) {
+    domUpdates.haveCookedRecipe(event);
+  } else if (event.target.classList.contains('bought-ingridients')) {
```

Comment:
Spelling check on ingredients on line 26. :)

Reply:
And done!